scanners/zap-automation-framework/examples/demo-nginx-automation-scan/scan.yaml

```
@@ -0,0 +1,77 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "zap-automation-nginx"
+data:
```

Config map linked to in the scan has a different name

```suggestion
  name: "zap-automation-framework-config"
```